include/crm/common/util.h

```diff
@@ -62,6 +62,9 @@ char *crm_strdup_printf(char const *format, ...) __attribute__ ((__format__ (__p
 int pcmk__parse_ll_range(const char *srcstring, long long *start, long long *end);
 gboolean pcmk__str_in_list(GList *lst, const gchar *s);
 
+bool pcmk__str_any_of(const char *s, ...) G_GNUC_NULL_TERMINATED;
+bool pcmk__str_none_of(const char *s, ...) G_GNUC_NULL_TERMINATED;
```

Comment:
Along with making these internal, can you move them (and the two above them) to common/internal.h?

Reply:
Yep, done.  That should have been in the last push in the first place.